badwords.csv

```diff
@@ -192,6 +192,7 @@ en,titties,breasts,1
 en,tit/tits,breasts (also a bird species) ,1
 en,tranny/trannies,vulgar term for transsexual,2
 en,turd/turds,poo,1
+en_US,ass/asses,literally buttocks but also used to mean a jerk (also a dated term for a donkey), 1
```

Comment:
i think the space after the comma might cause issues.

Reply:
Oops, addressed.